pyinfra/api/connectors/ansible.py

```diff
@@ -133,23 +143,46 @@ def _parse_ansible_hosts(hosts):
             yield host
 
 
-def parse_inventory_tree(inventory_tree, host_to_groups=dict(), group_stack=set()):
+def parse_inventory_tree(inventory_tree, host_to_groups=dict(), group_stack=set(), group_varz=dict()):
```

Comment:
Not spotted these before - these default mutable arguments need to be replaced (not that they are introduced by this PR)!